source/guides/routing/preventing-and-retrying-transitions.md

```diff
@@ -54,7 +54,7 @@ destination routes to abort attempted transitions.
 ```js
 App.DiscoRoute = Ember.Route.extend({
   beforeModel: function(transition) {
-    if (new Date() < new Date("January 1, 1980")) {
+    if (new Date() > new Date("January 1, 1980")) {
       alert("Sorry, you need a time machine to enter this route.");
```

Comment:
But if your date is before 1980, you get an error saying you need a time machine. Shouldn't it be the other way around?